internal/providers/terraform/ibm/testdata/is_instance_test/is_instance_test.tf

```diff
@@ -127,155 +127,174 @@ resource "ibm_is_dedicated_host_group" "dedicated_host_group" {
 
 locals {
   profiles = [
-    "cx2-2x4",
-    "cx2d-2x4",
-    "cx3d-2x5",
-    "cx3dc-2x5",
-    "bx2-2x8",
-    "bx2a-2x8",
-    "bx2d-2x8",
-    "bx3d-2x10",
-    "bx3dc-2x10",
-    "mx2-2x16",
-    "mx2d-2x16",
-    "ox2-2x16",
-    "mx3d-2x20",
-    "vx2d-2x28",
-    "ux2d-2x56",
-    "cx2-4x8",
-    "cx2d-4x8",
-    "cx3d-4x10",
-    "cx3dc-4x10",
-    "bx2-4x16",
-    "bx2a-4x16",
-    "bx2d-4x16",
-    "bx3d-4x20",
-    "bx3dc-4x20",
-    "mx2-4x32",
-    "mx2d-4x32",
-    "ox2-4x32",
-    "mx3d-4x40",
-    "vx2d-4x56",
-    "ux2d-4x112",
-    "cx2-8x16",
-    "cx2d-8x16",
-    "cx3d-8x20",
-    "cx3dc-8x20",
-    "bx2-8x32",
-    "bx2a-8x32",
-    "bx2d-8x32",
-    "bx3d-8x40",
-    "bx3dc-8x40",
-    "mx2-8x64",
-    "mx2d-8x64",
-    "ox2-8x64",
-    "gx2-8x64x1v100",
-    "mx3d-8x80",
-    "vx2d-8x112",
-    "ux2d-8x224",
-    "cx2-16x32",
-    "cx2d-16x32",
-    "cx3d-16x40",
-    "cx3dc-16x40",
-    "bx2-16x64",
-    "bx2a-16x64",
-    "bx2d-16x64",
-    "bx3d-16x80",
-    "gx3-16x80x1l4",
+
+    "bx2-128x512",
+    # "bx2-16x64",
```

Comment:
why are some profiles commented out?

Reply:
The only way I was able to get the unit tests running was to only to smaller batches at a time, so I'm leaving in one of each type... Are there any others in the list we should uncomment?